json/src/lib.rs

```diff
@@ -2161,7 +2162,18 @@ impl TryFrom<DMNStateDiffIntermediate> for DMNStateDiff {
                 })
             })
             .transpose()?;
-        let operator_payout_address = None; //todo
+        let operator_payout_address = operator_payout_address
```

Comment:
My first  impression is that processing of `operator_payout_address` is different than `payout_address` above (two  maps  here), copy-paste issue or expected behavior?

Reply:
My point is that:

* `payout_address` takes 9 lines of code, including one `map` operation
* `operator_payout_address` takes 12 lines of code, including two `map` operations

If they are the same type, I would expect both to take 9 lines of code and one `map` operation.

Reply:
Basically, there should be a helper function that you will use for both cases (and other similar, like `voting_address`?).